src/components/OTP.vue

```diff
@@ -145,6 +145,11 @@ export default {
   },
 
   computed: {
+    /** Retrieve phone number stored in vuex */
+    ...mapState({
+      statePhoneNumber: (state) => state.phoneNumber,
```

Comment:
```suggestion
      phoneNumber: (state) => state.phoneNumber,
```
This is also fine and much simpler

Reply:
It conflicts to a computed property, no?